packages/g/gleam/package.yml

```diff
@@ -0,0 +1,27 @@
+name       : gleam
+version    : 1.4.1
+release    : 1
+source     :
+    - https://github.com/gleam-lang/gleam/archive/refs/tags/v1.4.1.tar.gz : 1a3826d2d36f5442be34ec749895b03ed1538ef28e59b0134ff4001468f095f7
+homepage   : https://gleam.run
+license    : Apache-2.0
+component  : programming
+summary    : A friendly language for building type-safe systems that scale.
+description: |
+    Gleam is a friendly language for building type-safe systems that scale.
+    The power of a type system, the expressiveness of functional programming, and the reliability of the highly concurrent, fault tolerant Erlang runtime, with a familiar and modern syntax.
+    Gleam comes with compiler, build tool, formatter, editor integrations, and package manager all built in.
+    As part of the wider BEAM ecosystem, Gleam programs can use thousands of published packages, whether they are written in Gleam, Erlang, or Elixir.
+    Gleam can additionally compile to JavaScript, enabling you to use your code in the browser, or anywhere else JavaScript can run.
+networking : yes
+builddeps  :
+    rust
+rundeps    :
+    erlang
```

Comment:
`erlang` is already a rundep of `rebar3`, so it doesn't need to be in the list here.

Reply:
Also the syntax is wrong. Lists should like like this:

```yaml
rundeps    :
    - rebar3
```